canister/src/block_header_store.rs

```diff
@@ -85,3 +93,47 @@ fn init_block_headers() -> StableBTreeMap<BlockHash, BlockHeaderBlob, Memory> {
 fn init_block_heights() -> StableBTreeMap<u32, BlockHash, Memory> {
     StableBTreeMap::init(crate::memory::get_block_heights_memory())
 }
+
+#[cfg(test)]
+mod test {
+    use bitcoin::consensus::Encodable;
+    use proptest::proptest;
+
+    use crate::{block_header_store::BlockHeaderStore, test_utils::BlockBuilder};
+
+    #[test]
+    fn test_get_block_headers_in_range() {
+        let mut vec_headers = vec![];
```

Comment:
nit: `vec_` is redundant. It's clear that it's a vec

```suggestion
        let mut headers = vec![];
```